package.json

```diff
@@ -1,11 +1,23 @@
 {
   "name": "node-emoji",
-  "version": "1.10.0",
-  "description": "simple emoji support for node.js projects",
+  "version": "2.0.0",
+  "description": "Simple emoji support for Node.js.",
+  "repository": "https://github.com/omnidan/node-emoji.git",
   "author": "Daniel Bugl <daniel.bugl@touchlay.com>",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/omnidan/node-emoji.git"
+  "license": "MIT",
+  "main": "index.js",
+  "files": [
+    "index.js",
+    "index.d.ts"
```

Comment:
IME the three best ways to have TypeScript support are:

* Write the original package in TypeScript
* Include `.d.ts` files and tests in the repository
* Defer to [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) for typings

Including raw `.d.ts` files in this source repo but not testing them means contributors+maintainers will all have to manually check them. That's kind of hard if you're not TypeScript-familiar yourself -- and if you are, then why not write in TypeScript? 😜 

Proposal: remove `index.d.ts` and file a followup to discuss either rewriting in TypeScript or moving to DefinitelyTyped?

Reply:
I'm for rewriting it in TypeScript if you are up for it - that would definitely be the best solution 😁